packages/rspeedy/plugin-react-alias/src/index.ts

```diff
@@ -61,13 +79,17 @@ export function pluginReactAlias(options: Options): RsbuildPlugin {
           jsxDevRuntimeMainThread,
           reactLepusBackground,
           reactLepusMainThread,
+          reactCompat,
         ] = await Promise.all([
           resolve('@lynx-js/react/jsx-runtime'),
           resolve('@lynx-js/react/lepus/jsx-runtime'),
           resolve('@lynx-js/react/jsx-dev-runtime'),
           resolve('@lynx-js/react/lepus/jsx-dev-runtime'),
           resolve('@lynx-js/react'),
           resolve('@lynx-js/react/lepus'),
+          semver.gte(version, '0.112.0')
```

Comment:
Considering the canary version, it would be better to use `gt(version, '0.111.9999')`